pkg/utils/host/host.go

```diff
@@ -57,6 +58,10 @@ type Config struct {
 	AutoMountFilesystems bool
 }
 
+type ProcStat struct {
+	StartedAt int64
+}
```

Comment:
Do you foresee other fields we might want from procfs?
I wonder if it's necessary to have this struct since it is only one field. But if there are more later, then it is fine.

Reply:
I believe stuff like permissions and owner can be of interest but let me take that back to the team and verify.